src/Chainweb/Pact/PactService/ExecBlock.hs

```diff
@@ -388,6 +390,14 @@ applyPactCmds isGenesis env cmds miner mc blockGas txTimeLimit = do
       evalStateT (V.mapM (applyPactCmd isGenesis env miner txTimeLimit) cmds) (T2 mc blockGas)
     return txs
 
+-- we don't want pact tx failure or unknown exception messages when we're doing a replay.
+withSilentLoggerForReplays :: Logger logger => (logger -> PactServiceM logger tbl a) -> PactServiceM logger tbl a
+withSilentLoggerForReplays f = do
+  replaying <- view psReplaying
+  logger <- view psLogger
+  if replaying then L.withLoggerLevel L.Error logger f
```

Comment:
Not sure if this is too broad. This would also suppress unrelated legitimate warnings logs -- assuming that those could exist.